scripts/__tests__/articles-instrumentation.test.js

```
@@ -0,0 +1,141 @@
+    const localIdx = GA_BOOTSTRAP_SNIPPET.indexOf(
+      'localStorage.getItem(keys[i])',
+    );
+    expect(injectedIdx).toBeGreaterThan(-1);
+    expect(localIdx).toBeGreaterThan(injectedIdx);
+  });
+
+  test('mmkv 키는 dot/backslash 두 포맷을 모두 읽는다', () => {
+    // webpack alias 유무에 따라 키가 갈린다 — article-template 의 readSpaToken 과 같은 이유.
+    const keys = /var keys = (\[[^\]]*\]);/.exec(GA_BOOTSTRAP_SNIPPET)[1];
+    // eslint-disable-next-line no-eval -- 브라우저가 평가할 리터럴 그대로를 재현한다
+    expect(eval(keys)).toEqual([
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)